src/main/java/de/teamlapen/vampirism/entity/EntityVampirism.java

```diff
@@ -326,4 +344,18 @@ protected void updateAITasks() {
             onRandomTick();
         }
     }
+
+    /**
+     * gets a list of action combos and adding a random combo to the available actions of the IFaction entity
+     * 
+     * @param actionstmp
+     *            list of action combos
+     */
+    protected void setAvailableActions(List<IEntityAction[]> actionstmp) {
+        if (actionstmp.size() == 0)
+            return;
+        entityActions = new ArrayList<IEntityAction>();
+        for (IEntityAction e : actionstmp.get(this.getRNG().nextInt(actionstmp.size())))
```

Comment:
The code is not very straightforward here. Maybe you can untangle it a little bit and improve the documentation.
Maybe replacing the loop with an add all would be better: `entityActions.addAll(Arrays.asList(actionstmp.get(this.getRNG().nextInt(actionstmp.size()))));`

It adds all actions from one of the supplied arrays?
Currently all supplied arrays only contain one value

Reply:
actions used by classes are not finished